docs/release-notes/do-not-copy-resolv-conf.rst

```diff
@@ -0,0 +1,15 @@
+:Type: Software Installation
+:Summary: Do not copy installer /etc/resolv.conf to installed system
+
+:Description:
+    Anaconda does not copy /etc/resolv.conf file from installer environment to
+    the installed system anymore. Creating the file is a business of
+    systemd-resolved or NetworkManager and Anaconda is not going to interfere
```

Comment:
`systemd-resolved or NetworkManager. Anaconda is not going to interfere`
^
my recommendation

Reply:
Yeah, I was thinking about it too, but didn't bother because doc people will give it final touch. But here you are:)